TuviSRPLib/ProtonBase64Lib/ProtonBase64.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
+using System.Numerics;
+using System.Text;
+
+namespace ProtonBase64Lib
+{
+    public static class ProtonBase64
+    {
+        private const int BaseValue = 64;
+        private const int ByteSize = 8;
+        private const int SixBitsSize = 6;
```

Comment:
I would choose another name, probably with an explanation that we converting from base 2^8 to base 2^6, for example, the target unit size is equal to log2(64), something like that